src/providers/ThemeProvider.jsx

```diff
@@ -1,15 +1,31 @@
-import React, { createContext } from 'react';
-import useDarkMode from 'hooks/useDarkMode';
+import React, { createContext, useState } from "react";
+import useDarkMode from "hooks/useDarkMode";
 
 export const ThemeContext = createContext('light');
 
 export default ({ children }) => {
   const [theme, toggleTheme] = useDarkMode();
+  const [scrollPosition, setScrollPosition] = useState(0)
+
+  let scrolling = false;
+
+  document.addEventListener('scroll', () => {
+      scrolling = true;
+  },{ passive: true });
+
+  setInterval(() => {
+      if (scrolling) {
+          scrolling = false;
+          setScrollPosition(window.scrollY);
+          console.log(window.scrollY);
```

Comment:
It's a very miniscule thing but in general you should avoid leaving console logs in code. But this PR will be merged to my `develop` so i don't mind it so much for now. Thought i'd point that out ;)